docs/changes/11684.other.rst

```diff
@@ -0,0 +1 @@
+Use ``sphinx-astropy`` for ``numpydoc`` xref settings.
```

Comment:
I don't think this deserves a changelog entry, it's an internal change not visible by users.

Reply:
Done!